deepspeed/runtime/engine.py

```diff
@@ -1144,6 +1163,30 @@ def _configure_distributed_model(self, model):
         if self.sequence_parallel_size > 1:
             self.communication_data_type = self._config.seq_parallel_communication_data_type
 
+        
+        if self.sequence_parallel_size > 1:
+            log_dist(f"Using DS sequence parallelism (SP) of size {self.sequence_parallel_size}", ranks=[0])
+            #replace module attention with deespeed dist attention
+            #Assert we have torch version with device_mesh for parallel group
+            #Assert flash attn#
```

Comment:
Please remove the comments about asserts, the intent is clear from the asserts themselves.